numpy/core/src/multiarray/nditer_pywrap.c

```diff
@@ -1890,6 +1942,11 @@ static PyObject *npyiter_dtypes_get(NewNpyArrayIterObject *self)
         return NULL;
     }
 
+    if (self->open == 0) {
+        PyErr_SetString(PyExc_RuntimeError, "Iterator closed");
+        return NULL;
+    }
```

Comment:
This one feels totally safe to me

Reply:
fixing

Reply:
Ping

Reply:
fixed but there was a merge conflict, now fixed and merged